File: src/com/uipkg/simple/SplashScreen.java
package com.uipkg.simple;

import javax.swing.JFrame;

import com.uipkg.exceptions.NonApplicableException;

public class SplashScreen implements SimpleFrame {
  private float timeMS;
  public SplashScreen(float timeMS) {
    this.timeMS = timeMS;
  }
  @Override
  public void run() {
  }
  @Override
  public void runWithTask(Runnable s) throws NonApplicableException {
  }
  @Override
  public void killTask() throws NonApplicableException {
  }
  @Override
  public JFrame getInstanceFrame() {return null;
  }
  
  
}

File: src/com/uipkg/simple/SplashScreen.kt
package com.uipkg.simple

import com.uipkg.exceptions.NonApplicableException
import kotlin.Throws
import java.lang.Runnable
import javax.swing.JFrame

class SplashScreen(private val timeMS: Float) : SimpleFrame {
    override fun run() {}
    @Throws(NonApplicableException::class)
    override fun runWithTask(s: Runnable) {
    }

    @Throws(NonApplicableException::class)
    override fun killTask() {
    }

    override fun getInstanceFrame(): JFrame {
        return null
    }
}